.evergreen/config.yml

```diff
@@ -434,22 +441,23 @@ axes:
   - id: driver-versions
     display_name: Driver Version
     values:
-      - id: "lowest-supported"
-        display_name: "1.9.0"
-        variables:
-          DRIVER_VERSION: "1.9.0"
+      # TODO: Update matrix once v1.10 has been branched
```

Comment:
Disabling `latest-stable` would've required me changing a number of matrix entries below, which I wanted to avoid. I'll revisit and see how I can make this a little more resilient regarding such changes, since this typically happens during every version cycle.

Reply:
The only thing I came up with was renaming `latest-stable` to `latest` to account for it potentially being a dev version. I've held off on making the change but want to revisit this separately.

Reply:
I think the TODO comment can be revised. Before we branch `v1.10` (usually happens when we start the first work on v1.11), we can revert this to "stable" as soon as the 1.10 release ships.

Reply:
I've updated this. From the commit:
> * Add more detailed comment about re-enabling driver versions
> * Rename latest-stable to latest to account for varying stability
> * Prepare correct versions for 1.10-dev and 1.11-dev